src/commands/auth/login.js

```diff
@@ -78,17 +77,34 @@ The currently supported Adobe IMS login plugins are:
 * aio-lib-ims-jwt for JWT token based login supporting
   Adobe I/O Console service integrations.
 * aio-lib-ims-oauth for browser based OAuth2 login. This
-  plugin will launch a Chromium browser to guide the user through the
+  plugin will launch the default browser to guide the user through the
   login process. The plugin itself will *never* see the user's
   password but only receive the authorization token after the
-  user authenticated with Adobe IMS.
+  user has authenticated with Adobe IMS.
 `
 
 LoginCommand.flags = {
   ...ImsBaseCommand.flags,
-  force: flags.boolean({ char: 'f', description: 'Force logging in. This causes a forced logout on the context first and makes sure to not use any cached data when calling the plugin.', default: false }),
-  decode: flags.boolean({ char: 'd', description: 'Decode and display access token data' }),
-  bare: flags.boolean({ char: 'b', description: 'print access token only', default: false })
+  force: flags.boolean({
+    char: 'f',
+    description: 'Force logging in. This causes a forced logout on the context first and makes sure to not use any cached data when calling the plugin.',
+    default: false
+  }),
+  decode: flags.boolean({
+    char: 'd',
+    description: 'Decode and display access token data'
+  }),
+  bare: flags.boolean({
+    char: 'b',
+    description: 'print access token only',
+    default: false
+  }),
+  open: flags.boolean({
```

Comment:
@purplecabbage @shazron 
I wonder if it would make sense to ask the user to open the login page in the browser before ?
Something like this: 
<img width="379" alt="Screen Shot 2022-02-11 at 5 12 22 PM" src="https://user-images.githubusercontent.com/6012307/153627618-92fa93c7-73da-49d7-a16c-601c5ddf9ff7.png">